docs/assets/fhir-resource-ingestion-digestion-diagrams/bfd-pipeline-ccw-ingestion.plantuml

```diff
@@ -0,0 +1,32 @@
+@startuml
+title BFD-Pipeline-CCW Ingestion of Beneficiary/Claim Data Sequence
+participant ccw
+database s3
+participant "bfd-pipeline"
+participant splunk
+note over splunk: Specific Splunk Components: \n forwarders/indexers/search heads/ui/configuration
+participant slack
+database "bfd-db"
+ccw -> s3: Timestamped folder with: \n RIF files for a resource, manifest.xml \n placed in 'bfd-prod-etl' S3 bucket  
+s3 -> s3: Timestamped folder moved to \n 'Incoming' folder of bucket
+s3 -> splunk: New data discovered by splunk by quereying BFD-pipeline logs every 15 min
+splunk -> slack: Data discover status sent \n to #BFD-notices channel
+slack --> s3
+s3 -> "bfd-pipeline": ETL/RIF Loader process triggered 
```

Comment:
Are we intentionally glossing over the whole resource setup for the pipeline? (i.e. an instance is created, setup in the proper environment, etc) Is that encompassed by "process" vaguely here?